geomstats/geometry/dirichlet_distributions.py

```diff
@@ -1,5 +1,7 @@
 """Statistical Manifold of Dirichlet distributions with the Fisher metric."""
 
+import multiprocessing
```

Comment:
Can you try with joblib, as in [pairwise_dist](https://github.com/geomstats/geomstats/blob/544debcf81dc40d7e5d584c5606792c2cfa9f4eb/geomstats/geometry/riemannian_metric.py#L298) to be consistent?

Reply:
Searching the documentation and looking for examples, I haven't found a way to do this with joblib...